phpstan.neon

```diff
@@ -16,3 +16,7 @@ parameters:
         - packages/**/tests/**/Fixture/*
         - packages/**/tests/Fixtures/*
         - packages/fractor-rule-generator/templates
+    treatPhpDocTypesAsCertain: false
+    ignoreErrors:
+        - identifier: return.unusedType
+        - identifier: varTag.nativeType
```

Comment:
Why these ignores? I'm not exactly fond of ignoring whole classes of errors for the whole project…

Reply:
Could you please have a look at both of these and fix them?

Reply:
Will do, but in any case it would be better IMO to add just the current errors to a baseline instead of completely ignoring the error class.

Reply:
Some of the reported errors were pretty sensible, especially all those concerning the native types: they pointed out some inconsistencies in @var annotations that we had added in the past.

As for the "unusedType", those could have been ignored IMO, but it's better to have every method have a return type that actually specifies everything it will possibly return.